packages/authorization-process/test/api/getClientKeyById.test.ts

```diff
@@ -65,71 +62,56 @@ describe("API /clients/{clientId}/keys/{keyId} authorization test", () => {
     expect(res.status).toBe(403);
   });
 
-  it("Should return 404 for clientKeyNotFound", async () => {
-    authorizationService.getClientKeyById = vi
-      .fn()
-      .mockRejectedValue(clientKeyNotFound(mockKey1.kid, mockClient.id));
-
-    const res = await makeRequest(
-      generateToken(authRole.ADMIN_ROLE),
-      generateId(),
-      mockKey1.kid
-    );
-
-    expect(res.status).toBe(404);
-  });
-
-  it("Should return 404 for clientNotFound", async () => {
-    authorizationService.getClientKeyById = vi
-      .fn()
-      .mockRejectedValue(clientNotFound(mockClient.id));
-
-    const res = await makeRequest(
-      generateToken(authRole.ADMIN_ROLE),
-      generateId(),
-      mockKey1.kid
-    );
-
-    expect(res.status).toBe(404);
-  });
-
-  it("Should return 404 for clientKeyNotFound", async () => {
-    authorizationService.getClientKeyById = vi
-      .fn()
-      .mockRejectedValue(clientKeyNotFound(mockKey1.kid, mockClient.id));
-
-    const res = await makeRequest(
-      generateToken(authRole.ADMIN_ROLE),
-      generateId(),
-      mockKey1.kid
-    );
-
-    expect(res.status).toBe(404);
-  });
+  it.each([
+    {
+      name: "clientKeyNotFound",
+      error: clientKeyNotFound(mockKey1.kid, mockClient.id),
+      expectedStatus: 404,
+      clientId: generateId(),
+      keyId: mockKey1.kid,
+    },
+    {
+      name: "clientNotFound",
+      error: clientNotFound(mockClient.id),
+      expectedStatus: 404,
+      clientId: generateId(),
+      keyId: mockKey1.kid,
+    },
+    {
+      name: "organizationNotAllowedOnClient",
+      error: organizationNotAllowedOnClient(generateId(), mockClient.id),
+      expectedStatus: 403,
+      clientId: mockClient.id,
+      keyId: mockKey1.kid,
+    },
+    {
+      name: "securityUserNotMember",
+      error: securityUserNotMember(mockClient.users[0]),
+      expectedStatus: 403,
+      clientId: mockClient.id,
+      keyId: mockKey1.kid,
+    },
+  ])(
+    "Should return $expectedStatus for $name",
+    async ({ error, expectedStatus, clientId, keyId }) => {
```

Comment:
```suggestion
    async ({ error, expectedStatus, clientId }) => {
```
Since `keyId` is always the same it's not needed in the `it.each`

Reply:
Same, both `clientId` and `keyId` deleted